tequilapi/endpoints/list_identities_test.go

```diff
@@ -0,0 +1,38 @@
+package endpoints
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListIdentities(t *testing.T) {
+
+	req := httptest.NewRequest("GET", "/irrelevant", nil)
+	resp := httptest.NewRecorder()
+
+	mockIdm := newManager("0x000000000000000000000000000000000000000A")
+	handlerFunc := ListIdentitiesEndpointFactory(mockIdm).ListIdentities
```

Comment:
Factory could return `Handler` callback

Reply:
factory now returns an object with `Get` handler.
I think we might have a pattern where we'd have a single object for each endpoint with `Get` `Put` `Delete` handlers
let me know if you still want a factory that returns single callbacks for each endpoint method